server/bin/pbench-server-prep-shim-002.py

```diff
@@ -241,13 +241,16 @@ def process_tb(config, logger, receive_dir, qdir_md5, duplicates, errors):
             nerrs += 1
             continue
 
-        # Restore the SELinux context properly
-        try:
-            selinux.restorecon(dest / tb.name)
-            selinux.restorecon(dest / tbmd5.name)
-        except Exception as e:
-            # log it but do not abort
-            logger.error("{}: Error: 'restorecon {}', {}", config.TS, dest / tb.name, e)
+        if selinux.is_selinux_enabled():
+            # Restore the SELinux context properly
+            try:
+                selinux.restorecon(dest / tb.name)
+                selinux.restorecon(dest / tbmd5.name)
+            except Exception as e:
+                # log it but do not abort
+                logger.error(
+                    "{}: Error: 'restorecon {}', {}", config.TS, dest / tb.name, e
+                )
```

Comment:
Ha; you've broken the legacy gold files by removing these errors from the log.

Reply:
The update to the mock `selinux` module fixed this.  😁